src/scores/blosum62.rs

```diff
@@ -66,13 +66,13 @@ lazy_static! {
 #[inline]
 fn lookup(a: u8) -> usize {
     if a == b'Y' {
-        23 as usize
+        23_usize
```

Comment:
I wonder if these are needed at all, since the return type is `usize`, i.e. this can be inferred quite easily.

Reply:
Good eye and right you are! I removed them.